src/app/communities/CommunityItem.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { describe, expect, test, vi, beforeEach, afterEach } from 'vitest'
+import { CommunityItem } from './CommunityItem'
+import { cleanup, render } from '@testing-library/react'
+import { BoostData, useNFTBoosterContext } from '@/app/providers/NFT/BoosterContext'
+
+vi.mock('@/app/providers/NFT/BoosterContext', () => ({
+  useNFTBoosterContext: vi.fn(),
+}))
+
+describe('CommunityItem', () => {
+  const communityItemProps = {
+    leftImageSrc: '/src',
+    title: 'Early adopters',
+    subtitle: '',
+    nftAddress: '0x123',
```

Comment:
This could be stored in a const and be used below also

Reply:
updated